cfme/v2v/migration_plans.py

```diff
@@ -417,6 +417,13 @@ def _in_progress():
                             format(plan_name=self.name, visibility=is_plan_visible,
                                    message=new_msg)
                         )
+                        h, m, s = str(plan_time_elapsed).split(":")
+                        if (60 * wait_time) < (int(h) * 3600 + int(m) * 60 + int(s)):
```

Comment:
Optional: I would prefer to use https://docs.python.org/3/library/datetime.html#datetime.timedelta as I think it is easy to use and comprehensive:
```
In [12]: timedelta(minutes=30) < timedelta(seconds=30*60+1) < timedelta(minutes=30, seconds=2)                                                                                                
```

Reply:
Thank you for the suggestion.